Sources/XcodeGenKit/FileWriter.swift

```diff
@@ -41,6 +41,33 @@ public class FileWriter {
         }
     }
 
+    public func writeHook(at path: Path) throws {
+        var content = ""
+        if path.exists, let existingHook: String = try? path.read() {
+            content = existingHook
+        }
+
+        // Add XcodeGen in the git hook if not exist
+        // Checking command existence fixes command not found issue when using SourceTree
+        let comment = "# XcodeGen auto generated"
+        if !content.contains(comment) {
+            content += """
+            \(comment)
+            if ! [ -x "$(command -v xcodegen)" ]; then
```

Comment:
We may simplify the script by calling xcodegen directly from `/usr/local/bin/xcodegen`